ros2_controllers/CMakeLists.txt

```
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.5)
+cmake_minimum_required(VERSION 3.16)
 project(ros2_controllers)
```

```suggestion
project(ros2_controllers LANGUAGES NONE)
```
This will save a little bit of time during configuration.